weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CustomGridTreeDataGroupingCell.tsx

```diff
@@ -202,7 +202,7 @@ export const CustomGridTreeDataGroupingCell: FC<
             </Box>
             {call?.rawSpan?.summary && (
               <TraceUsageStats
-                usage={call.rawSpan.summary.usage}
+                costData={call.rawSpan.summary.costs}
```

Comment:
i'd recommend converting this to `call.traceCall.summary` (see the typing comment below)